src/pages/organizers/[organizerId]/ownerships/index.page.tsx

```diff
@@ -1,31 +1,49 @@
 import groupBy from 'lodash/groupBy';
 import { useRouter } from 'next/router';
-import { useMemo } from 'react';
-import { useTranslation } from 'react-i18next';
-import { dehydrate } from 'react-query';
+import { useMemo, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { dehydrate, useQueryClient } from 'react-query';
 
 import { useGetOrganizerByIdQuery } from '@/hooks/api/organizers';
 import {
   OwnershipRequest,
   RequestState,
+  useApproveOwnershipRequestMutation,
   useGetOwnershipRequestsQuery,
+  useRejectOwnershipRequestMutation,
 } from '@/hooks/api/ownerships';
 import { Organizer } from '@/types/Organizer';
+import { Values } from '@/types/Values';
 import { Alert, AlertVariants } from '@/ui/Alert';
+import { Box } from '@/ui/Box';
 import { Button, ButtonVariants } from '@/ui/Button';
 import { Icon } from '@/ui/Icon';
 import { Icons } from '@/ui/Icon';
 import { Inline } from '@/ui/Inline';
+import { Modal, ModalSizes, ModalVariants } from '@/ui/Modal';
 import { Page } from '@/ui/Page';
 import { Stack } from '@/ui/Stack';
 import { Title } from '@/ui/Title';
 import { getApplicationServerSideProps } from '@/utils/getApplicationServerSideProps';
 
 import { OwnershipsTable } from './OwnershipsTable';
 
+const ActionType = {
+  APPROVE: 'approve',
+  REJECT: 'reject',
+} as const;
+
+type ActionType = Values<typeof ActionType>;
+
 const Ownership = () => {
   const router = useRouter();
   const { t, i18n } = useTranslation();
+  const queryClient = useQueryClient();
+  const [selectedRequest, setSelectedRequest] = useState<OwnershipRequest>();
+  const [isQuestionModalVisible, setIsQuestionModalVisible] = useState(false);
+  const [actionType, setActionType] = useState<ActionType>();
+  const [isMutationSuccessful, setIsMutationSuccessful] = useState(false);
```

Comment:
I would call it `isSuccessAlertVisible`  or something similar, since when you do `setIsMutationSuccessful(false)` doesn't mean the mutation failed, but that the alert can be hidden